andela/Ah-frontend-guardians
# #163857118 Deploy to heroku

#### What does this PR do?
This PR enables automatic deploys to Heroku whenever a PR is merged.

#### Description of Task to be completed?
* setup a server that listens to requests
* add Procfile

#### How should this be manually tested?
Visit the staging app here: https://ah-frontend-guardians.herokuapp.com/

*Expected output: *
You should see the text below
"Hello World"

#### What are the relevant pivotal tracker stories?
[#163857118](https://www.pivotaltracker.com/story/show/163857118)

#### Additional Context:
Once this PR gets merged to develop, I will be able to enable review apps on heroku and every PR created will be able to create it's own review app.
